src/pages/admin/reports/components/funds-expenditures-section/FundsExpendituresSection.module.scss

```
@@ -26,15 +26,15 @@
 
-.disclaimer-header {
+.disclaimer-top-row {
     display: flex;
     align-items: center;
     justify-content: space-between;
 }
 
 .disclaimer-label {
     @include type.subtitle-1-medium;
-    font-weight: 700;
+    font-weight: 500;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)